src/main/resources/templates/index.html

```diff
@@ -7,50 +7,42 @@
     <meta name="author" content="Wesley Reed">
     <title>Inventory Manager</title>
 
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <!--Boostrap core CSS end -->
+
+    <nav th:replace="fragments/inlineStyles :: styles"></nav>
+</head>
+<body>
+
+    <nav th:replace="fragments/navigation :: navbar"></nav>
 
-    <!--Dan's stuff-->
-    <p th:text="${item}" />
     <form action="#" th:action="@{/saveItem}" th:object="${item}" method="GET">
         <label>Item name:</label>
-        <input type="text" name="name" th:field="*{name}"/><br><br>
+        <input type="text" name="name" th:placeholder="*{getItemName}"/><br><br>
```

Comment:
replaced the Thymeleaf field tags with placeholder tags. This looked like the implementation you guys were going for so I wanted to provide an alternative to pre-filling the inputs.